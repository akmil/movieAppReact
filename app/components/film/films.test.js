import React from 'react';
// import ReactDOM from 'react-dom';
// import renderer from 'react-test-renderer';
import FilmsBox from './filmsBox';
import { shallow } from 'enzyme';

describe('FilmsBox', () => {
  it('should render without errors', () => {
    // const wrapper = renderer.create(<MainLayout/>)
    const wrapper = shallow(<FilmsBox/>);
    expect(wrapper).toMatchSnapshot();
  });
});
