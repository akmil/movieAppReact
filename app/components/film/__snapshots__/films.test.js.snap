// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render without errors 1`] = `
<div
  className="films films-box-wrapper"
>
  <div
    className="films-box-header"
  >
    <div
      className="logo col-33"
    >
      netfixroulette
    </div>
    <FilmsSearchButton
      name="FilmsSearchButton"
      theme="white"
    />
    <div
      className="d-flex"
    >
      <FilmsCover
        src="https://images.pexels.com/photos/584302/pexels-photo-584302.jpeg?auto=compress&cs=tinysrgb&h=350"
      />
      <div
        className="films-box-body"
      >
        <FilmsTitle
          activeClass="activeClass"
          rate={8.2}
          title="Pulp fiction"
        />
        <div
          className="mb-3"
        >
          <span
            className="mr-3 release"
          >
            1995
          </span>
          <span
            className="ml-3 duration"
          >
            100
          </span>
        </div>
        <FilmsDescription
          description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
        />
      </div>
    </div>
  </div>
  <div
    className="results results-wrapper"
  >
    <Connect(FilmsSearchButton)
      filter="SHOW_DRAMA"
      name="show Drama"
      theme="white"
    />
    <Connect(FilmsSearchButton)
      filter="SHOW_COMEDY"
      name="show Comedy"
      theme="white"
    />
    <Connect(FilmsSearchButton)
      filter="SHOW_COMEDY"
      name="fetch"
      theme="white"
    />
  </div>
  <FilmsList />
</div>
`;
